electron-app/tests/e2e/pages/account-balances-page.ts

```diff
@@ -27,6 +27,43 @@ export class AccountBalancesPage {
       .should('have.length', visible + 1);
   }
 
+  balanceShouldMatch(balances: ApiManualBalance[]) {
+    let i = 0;
+    for (const balance of balances) {
+      cy.get('.manual-balances-list tbody').find('tr').eq(i).as('row');
+
+      cy.get('@row')
+        .find('.manual-balances-list__amount')
+        .should('contain', bigNumberify(balance.amount).toFormat(2));
+
+      i += 1;
+    }
+
+    // cy.get('.manual-balances-list tbody').find('tr').eq(position).as('row');
```

Comment:
If we are not using these anymore then it would be better to remove them completely rather than leaving commented code in. If we need them in the future we still have the git history.